csp_billing_adapter/bill_utils.py

```diff
@@ -48,17 +53,25 @@ def get_max_usage(metric: str, usage_records: list) -> int:
     if not usage_records:
         return 0
 
-    max_usage = max(record.get(metric, 0) for record in usage_records)
+    max_usage = max(
+        record.get(metric, 0) for record in usage_records
```

Comment:
This pair of lines should be moved to a helper function since it is duplicated in get_average_usage() and it would be better to maintain it only once.